universum/analyzers/scan_build_report.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+from lxml import etree
+import sys
+
+from typing import List
+
+from . import utils
+
+
+def main() -> int:
+    settings = scan_build_report_argument_parser().parse_args()
+    files = utils.expand_files_argument(settings)
+    return utils.report_parsed_outcome(None,
+                                       lambda _: scan_build_report_output_parser(files),
+                                       settings.result_file)
+
+
+def scan_build_report_argument_parser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser(description="Parse scan-build report")
+    utils.add_files_argument(parser)
+    utils.add_result_file_argument(parser)
+    return parser
+
+
+def scan_build_report_output_parser(file_list: List[str]) -> List[utils.ReportData]:
+    result: List[utils.ReportData] = []
+    for report_file in file_list:
+        with open(report_file) as f:
+            parser = etree.HTMLParser()
+            tree: etree.ElementTree = etree.parse(f, parser)
+            found = any(x.text for x in tree.iter() if x.text == 'Bug Summary')
+            if not found:
+                continue
```

Comment:
this is to filter out non-report html files - these files don't indicate a problem